src/main/java/com/ericsson/ei/EndpointSecurity.java

```diff
@@ -85,8 +92,19 @@ private String decodeBase64(String password) {
     }
 
     private void addLDAPServersFromList(JSONArray serverList, AuthenticationManagerBuilder auth) throws Exception {
+        TextFormatter textFormatter = new TextFormatter();
+        StandardPBEStringEncryptor encryptor = new StandardPBEStringEncryptor();
+        encryptor.setPassword(jasyptEncryptorPassword);
         for (int i = 0; i < serverList.length(); i++) {
             JSONObject server = (JSONObject) serverList.get(i);
+            String password = server.getString("password");
+            if (password.startsWith("ENC(") && password.endsWith(")")) {
```

Comment:
Please break this out to its own method with descriptive name :)

Reply:
Done.